Library/Homebrew/test/rubocops/lines_spec.rb

```diff
@@ -782,5 +782,16 @@ class Foo < Formula
         end
       RUBY
     end
+
+    it "reports an offense when using depends_on \"mpich\"" do
+      expect_offense(<<~RUBY)
```

Comment:
```suggestion
      expect_offense(<<~RUBY, "/homebrew-core/")
```

Reply:
@reitermarkus you rock! Thanks, that fixed it!

Should I break this back out into its own class again if I want to provide auto-correct functionality? I can add a test for that too, it looks like, now that I understand how that works. I think if I leave it in the miscellaneous class, which currently has no auto-correct, the auto-correct won't be specific to just the MPI lib choice.

Reply:
Yeah, I think a separate class for each auto-correctable cop is preferable.

Reply:
Can I keep it in lines.rb or should I make a new file/module?

Reply:
@reitermarkus why does your change resolve the problem?

I was looking for documentation on `expect_offense()` and came up with https://www.rubydoc.info/gems/rubocop/RuboCop%2FRSpec%2FExpectOffense:expect_offense

The second argument, `file` is not well documented there, IMO. Is that argument taken to be a hypothetical path to the source?